### ISSUE TITLE
view side by side vertically

### ISSUE THREAD

**Issue by yofriadi** [sindrets/diffview.nvim#24]
Hi, thanks for the plugin.
Is there any config to make itu side by side vertically top and bottom instead?
I have a trouble since my screen isn't that wide for side by side left and right
thank you

**Comment by sindrets** [comment]
No, this is not possible at the moment. But I've been planning to make it respect your `diffopt` more, so that you can set it to be either `horizontal` or `vertical` like you normally would with vimdiff.

**Comment by sindrets** [comment]
@yofriadi you can now make the diff windows align vertically by adding `vertical` to your `'diffopt'` before you open the diffview. Example:

```vim
set diffopt+=vertical
```

**Comment by sindrets** [comment]
@yofriadi just want to let you know that I made a mistake when implementing this, that I have now corrected, but it means you have to change your `diffopt` setting. `diffopt`'s `horizontal` and `vertical` refers to split direction, and not layout direction, so for it to be consistent with the `:diffsplit` command I have now corrected Diffview to treat these settings as such.

This means you have to `set diffopt+=horizontal` for a horizontal split / vertical layout.
